src/goto-checker/solver_factory.cpp

```diff
@@ -297,7 +297,8 @@ get_sat_solver(message_handlert &message_handler, const optionst &options)
     else if(solver_option == "cadical")
     {
 #if defined SATCHECK_CADICAL
-      return make_satcheck_prop<satcheck_cadicalt>(message_handler, options);
+      return make_satcheck_prop<satcheck_cadical_no_preprocessingt>(
```

Comment:
❓ Does this need to honour the `no_simplifier` flag as for the other solvers above?

Reply:
Once the preprocessing becomes default, it should.